app/templates/model-details.html

```diff
@@ -9,13 +9,13 @@
       <div class="panel panel-default">
 
         <div class="panel-heading">
-          <h2 class="panel-title">{{activeModel.data.name}}</h2>
```

Comment:
Because we are now lazy loading the model details. I had two options:

1. Create a fully instantiated model at mount with empty strings such data will not be undefined at that point
2. Make a get-method that cleanly handles an undefined model

Given the event-driven and reactive nature of Vue, the latter seemed more robust: if a component anywhere decides to destroy the activeModel, this component will never throw (a huge amount) of errors.